script/doctest-output.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
This seems unrelated, but sure, I can see it being useful.

Reply:
I slipped it in instead of creating a new PR. I've needed it during doc development.